package-scripts.js

```diff
@@ -95,8 +95,8 @@ module.exports = {
       default: npsUtils.series.nps("lint", "test")
     },
     watch: {
-      es: "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=es babel src --out-dir es --config-file ../../.babelrc.js --copy-files --extensions .tsx,.ts,.jsx,.js --watch",
-      lib: "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=commonjs babel src --out-dir lib --config-file ../../.babelrc.js --copy-files --extensions .tsx,.ts,.jsx,.js --watch",
+      es: "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=es babel src --out-dir es --config-file ../../.babelrc.build.js --copy-files --no-copy-ignored --extensions .tsx,.ts,.jsx,.js --watch",
+      lib: "lerna exec --parallel --ignore victory-native --ignore victory-vendor -- cross-env BABEL_ENV=commonjs babel src --out-dir lib --config-file ../../.babelrc.build.js --copy-files --no-copy-ignored --extensions .tsx,.ts,.jsx,.js --watch",
```

Comment:
Here is the tricky thing I ran into - we want to copy `.d.ts` files into the built output directories, but NOT test files. The `--copy-files` CLI argument copies all un-transformed files into the built directories, regardless of whether or not they are in the list of ignored files. So that is why we can ignore transforming `.d.ts` files in our base babel config, but they still get copied into the `es` and `lib` directories. In the latest babel version, they added the `--no-copy-ignored` flag to handle this case.

However, we do NOT want to ignore test files universally, because we use the same config for `babel-jest`. This was my reasoning for adding a separate babel build config that ignores tests, but the default babel config does not. 

The other possibility here is ignoring test files via a CLI `--ignore` flag, but the babel CLI does not support file extension matching, and we could need to add separate `--ignore` args for each file extension. The babelrc file supports regex matching to make this easier.

Reply:
That seems fine to me. I'll note that we would like to avoid babel-ing test files for speed of transpilation, _but_ if we want to have a scheme that copies to `es` and `lib` we can just adjust `packages/*/.npmigore` to exclude them from publishing....

Reply:
I wanted to exclude them from npm, and also avoid running each test three times in the test runner without specifically excluding `es` and `lib` from the test command.